### PR TITLE
Include the unit of measure for size field in Amazon S3

### PR DESCRIPTION
**Pull request by akanshi-crest**
Added bytes as measurement unit for the field `size` as mentioned in the issue https://github.com/elastic/connectors-python/issues/363

#### Pre-Review Checklist
- [x] Covered the changes with automated tests
- [x] Tested the changes locally
- [x] Added a label for each target release version (example: `v7.13.2`, `v7.14.0`, `v8.0.0`)
- [x] Considered corresponding documentation changes
- [x] Contributed any configuration settings changes to the configuration reference

### CODE REVIEW

**Comment by tarekziade** [comment]
for s3 changes please add the `v8.6.0` label so it's backported to the `8.6` branch

File: connectors/sources/s3.py [hunk 1]
```diff
@@ -185,19 +186,23 @@ async def get_docs(self):
                         doc_id = md5(
                             f"{bucket}/{obj_summary.key}".encode("utf8")
                         ).hexdigest()
+                        object_size = await obj_summary.size
 
                         doc = {
                             "_id": doc_id,
                             "filename": obj_summary.key,
-                            "size": await obj_summary.size,
+                            "size": f"{object_size} bytes",
```
Comment (tarekziade) [inline connectors/sources/s3.py:194]: this turns the field into a string, is this what we really want @ppf2 ?

Comment (ppf2) [reply connectors/sources/s3.py:194]: Certainly not :) We don't want to change the field into a string.  `size` should remain an integer/long field to support sorting and mathematical functions.  We simply need to provide an indicator of the unit of measure, not to change the actual value of the field (e.g. either update the field name to include bytes there, or add a tooltip to the UI for `size` to tell the user that it is in bytes, etc..

Comment (akanshi-crest) [reply connectors/sources/s3.py:194]: If we are planning to a tooltip for this so it is going to be handle from the UI level and may be @tarekziade can handle this thing but if we have to update field name to include bytes in that case can you please suggest what's the most suitable name for this.
example: **size (in bytes)**, **size (bytes)**

Comment (akanshi-crest) [reply connectors/sources/s3.py:194]: Please let us know if you want me to update the size key name to size (bytes) or anything else. However if we are planning to add a tooltip, It needs to be taken care at the UI level.
cc Tarek

Comment (ppf2) [reply connectors/sources/s3.py:194]: +@danajuratoni This feels like a PM decision on the design.

Comment (danajuratoni) [reply connectors/sources/s3.py:194]: Here's my delayed (sorry for the delay) comment on wether "bytes" should be in the field name (to include bytes) or a tooltip in UI. 
Atm the only feasible option is to change the field name to include bytes, there is no tooltip capability in 8.7. We'll want to add that in 8.8, but for now we should use the field name.

Comment (akanshi-crest) [reply connectors/sources/s3.py]: I have updated the PR

File: connectors/sources/s3.py [hunk 2]
```diff
@@ -110,7 +110,7 @@ async def _get_content(self, doc, region, timestamp=None, doit=None):
         if os.path.splitext(filename)[-1] not in TIKA_SUPPORTED_FILETYPES:
             logger.debug(f"{filename} can't be extracted")
             return
-        if doc["size"] > DEFAULT_MAX_FILE_SIZE:
+        if doc["size (bytes)"] > DEFAULT_MAX_FILE_SIZE:
```
Comment (tarekziade) [inline connectors/sources/s3.py:113]: let's go for `size_in_bytes` -- I don't want to introduce a filed name that's a complex description

**Review by tarekziade — CHANGES_REQUESTED** [review]

**Review by tarekziade — APPROVED** [review]

**Pull request merged**